docs/changelog.md

```diff
@@ -88,10 +95,10 @@ permalink: /changelog/
   * 🌟 Added support for Glean crash pings in the `GleanCrashReporterService`. [Bug 1810951](https://bugzilla.mozilla.org/show_bug.cgi?id=1810951)
 
 # 110.0.0
-* [Commits](https://github.com/mozilla-mobile/firefox-android/compare/v109.0.0...v110.0.0)
```

Comment:
We stopped getting these types of `v<version>.0.0` tags after 108. There wasn't a consistent tagging of an exact `<version>.0.0` version so the next best thing to do here is to just reference the branch instead of the tags.

This can be examined by grepping the tags from https://github.com/mozilla-mobile/firefox-android/tags and looking for a particular version.

For example, the tags for 109 were the following:
components-v109.0b3
components-v109.0b4
components-v109.0b5
components-v109.0b6
components-v109.1.1
components-v109.2.0

110 were the following:
components-v110.0.1
components-v110.0b1
components-v110.0b2
components-v110.0b3
components-v110.0b4
components-v110.0b5